src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <ProjectGuid>{5F553243-042C-45C0-8E49-C739131E11C3}</ProjectGuid>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
+    <DefineConstants Condition="'$(TargetsNETStandard)' != 'true'">$(DefineConstants);BUILDING_INBOX_LIBRARY</DefineConstants>
```

Comment:
nit: I don't think using `BUILDING_INBOX_LIBRARY` as the constant name makes sense within the test project.

Maybe we can invert the condition here and rename this to: `NETSTANDARD`.
https://github.com/dotnet/corefx/blob/e23119d577e644d2c2a25419c88c1181681358e0/src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj#L7

Or something like this (`SINGLE_TO_INT32_BITS_AVAILABLE`), but that is too specific:
https://github.com/dotnet/corefx/blob/0110344cd384bcfaffed2385cd9df87d956bd68b/src/System.IO.Compression.Brotli/tests/System.IO.Compression.Brotli.Tests.csproj#L4